volatility3/cli/volshell/windows.py

```diff
@@ -44,11 +44,63 @@ def list_processes(self):
             )
         )
 
+    def get_process(self, pid=None, v_offset=None, p_offset=None):
+        """Returns the EPROCESS object that matches the pid, if v_off/p_off is provided, construct the _EPROCESS object and return that instead"""
+
+        kvo = self.config[interfaces.configuration.path_join("kernel", "offset")]
+        layer_name = self.current_layer
+        symbol_table = self.current_symbol_table
+        memory = self.context.layers[layer_name]
+
+        if p_offset is not None and v_offset is None:
+            # TODO
+            raise NotImplementedError
+
+        if v_offset is not None:
+            # v_off takes precedence, if pid is supplied do a sanity check and warn if not matched
+            ntkrnlmp = self.context.module(
+                symbol_table, layer_name=layer_name, offset=kvo
```

Comment:
The layer is being encoded in module you're creating here, but you can just call `context.object` providing the `symbol_table + constants.BANG + "_EPROCESS"`...